### DISCUSSION TITLE
在RangePicker中，如何实现结束时间不能小于开始时间？

### DISCUSSION THREAD

**Discussion by MattinaYang** [ant-design/ant-design#50093] [Q&A]
RangePicker没有这个限制，当结束时间小于开始时间时，会自动将两者换位；但我想实现选择开始时间后，将开始时间之前的时间置为不可选。想请教下如何实现？

**Comment by Zyf665** [comment]
你要的是这个效果吗？ https://stackblitz.com/edit/vitejs-vite-qhwud9?file=src%2FApp.tsx

**Reply by MattinaYang** [reply]
不是。比如我先选了结束时间为8.6，我希望接下来选开始时间时，8.6以后的时间均不可选择
![image](https://github.com/user-attachments/assets/acdc79b3-ac87-4244-b088-76d9c0bd2238)
